testutil/checkers.go

```diff
@@ -83,3 +83,61 @@ func (c *containsChecker) Check(params []interface{}, names []string) (result bo
 		return false, fmt.Sprintf("%T is not a supported container", container)
 	}
 }
+
+type deepContainsChecker struct {
+	*check.CheckerInfo
+}
+
+// DeepContains is a Checker that looks for a elem in a container using
+// DeepEqual.  The elem can be any object. The container can be an array, slice
+// or string.
+var DeepContains check.Checker = &deepContainsChecker{
+	&check.CheckerInfo{Name: "DeepContains", Params: []string{"container", "elem"}},
+}
+
+func (c *deepContainsChecker) Check(params []interface{}, names []string) (result bool, error string) {
+	defer func() {
+		if v := recover(); v != nil {
```

Comment:
Ah, I see the point of the panic recovery in the original. Originally I just copied that from check without much consideration. I will refactor this and re-push.